dist/chart-legend/chart-legend.css

```diff
@@ -0,0 +1,103 @@
+.chart-legend {
+  display: inline-block;
+}
+.chart-legend [hidden] {
```

Comment:
This seems more like you are trying to clip text for assistive technology rather than hide it completely for all users?

Reply:
correct, this is so that a percentage is read out by the screen reader.  Not sure if we want to display the percentage, but I know it was mentioned we should have it for the screen readers.

Reply:
Okay then, in that case we shouldn't use the hidden attribute. As the intention of hidden is to hide for all users.